pandas/core/generic.py

```diff
@@ -9953,11 +9953,24 @@ def _check_percentile(self, q):
         """
 
     @Appender(_shared_docs['pct_change'] % _shared_doc_kwargs)
-    def pct_change(self, periods=1, fill_method='pad', limit=None, freq=None,
-                   **kwargs):
+    def pct_change(self, periods=1, fill_method=None, limit=None, freq=None,
+                   skipna=None, **kwargs):
+        if skipna and fill_method is not None:
+            raise ValueError("cannot pass both skipna and fill_method")
+        elif skipna and limit is not None:
```

Comment:
do you have tests for each of these conditions?

Reply:
Yes, `test_pct_change_skipna_raises` in tests/generic/test_generic.py.